std/array.d

```diff
@@ -3467,16 +3467,13 @@ if (isDynamicArray!A)
     }
 
     /**
-     * Appends `rhs` to the managed array.
+     * Appends to the managed array.
      * Params:
      *     op = the assignment operator `~`
```

Comment:
I think that this parameter would not have to be documented because the "colon value" (which appears in the generated docs) 
- is self explanatory 
- cannot change.